modules/docs/markdown/06-guides/extract-request-info.md

```diff
@@ -100,6 +65,7 @@ Note that it is getting the `RequestInfo` from the `IO[RequestInfo] that is bein
 will be created using the same `IOLocal` instance is passed to our middleware implementation.
 That way, the middleware can set the `RequestInfo` value that we are reading here.
 
+[[generalisation.md]]
```

Comment:
what _is_ this syntax

Reply:
doesn't seem right.

Also the `IO[RequestInfo]` snippet above isn't closed correctly:

<img width="1074" alt="image" src="https://user-images.githubusercontent.com/894884/200921831-e92a3377-3170-499a-ad86-98d9da59f847.png">

Reply:
Fixed the above in 5017f06ae8125dfa5587cf4f6f5163c5808e9243 but I'm not sure what you wanted to do with this line here so I won't be changing it

Reply:
bd98c94